bin/capsule.js

```diff
@@ -869,11 +888,59 @@ const addFilesToS3Bucket = async (projectName, bucketName) => {
   });
 }
 
+/**
+ *
+ * @method getCloudFrontDistID
+ *
+ * @param {String} bucketName
+ *
+ * @return {String} distId
+ *
+ */
+const getCloudFrontDistID = async (bucketName) => {
+  let params = {};
+  return new Promise((resolve, reject) => {
+    cfr.listDistributions(params, function(err, data) {
+      if (err) {
+        logIfVerbose(`${err} , ${err.stack}`);
+        reject(undefined)
```

Comment:
The reject should always return a descriptive error. In this case it should be reject(err);

Reply:
Sounds good we'll add a fix to `stylistic_fixes`